files/zh-cn/web/css/font-family/index.md

```diff
@@ -15,7 +15,7 @@ CSS 属性 `font-family` 允许您通过给定一个有先后顺序的，由字
 
 应当至少在使用的 `font-family` 列表中添加一个通用的字体族名，因为无法保证用户的计算机内已经安装了指定的字体，也不能保证使用 {{cssxref("@font-face")}} 提供的字体移动能够正确地下载。提供通用的字体族使得浏览器可以在无法得到最佳字体的情况下使用一个相对接近的备选字体。
 
-`font-family` 属性指定的是一个优先级从高到低的可选字体列表。字体的选定**不是**在发现用户计算机上安装的列表中的第一个字体时停止。相反，对字体的选择是**逐字**进行的。也就是说即使某个字符周围都在某个字体中可以显示，但该字符在当前的字体文件中没有适合的图形，那么会继续尝试列表中靠后的字体。（不过这在 Internet Explorer 6 以及之前的版本的 IE 浏览器中不适用。）当一个字体只在某些特定的 [font-style](/zh-CN/docs/CSS/font-style)、 [font-variant](/zh-CN/docs/CSS/font-variant)、或 [font-size](/zh-CN/docs/CSS/font-size) 属性值下有效时，这些属性的值也可能对字体族 font family 的选择造成影响。
+`font-family` 属性指定的是一个优先级从高到低的可选字体列表。字体的选定**不是**在发现用户计算机上安装的列表中的第一个字体时停止。相反，对字体的选择是**逐字**进行的。也就是说即使某个字符周围都在某个字体中可以显示，但该字符在当前的字体文件中没有适合的图形，那么会继续尝试列表中靠后的字体。（不过这在 Internet Explorer 6 以及之前的版本的 IE 浏览器中不适用。）当一个字体只在某些特定的 {{cssxref("font-style")}}、{{cssxref("font-variant")}} 或 {{cssxref("font-size")}} 属性值下有效时，这些属性的值也可能对字体族 font family 的选择造成影响。
```

Comment:
Let's use the same writing style as the [en-US document](https://github.com/mdn/content/blob/74d02a842f4d369703c4f8a27a46c4a7bb1498c7/files/en-us/web/css/font-family/index.md?plain=1#L20)

```suggestion
`font-family` 属性指定的是一个优先级从高到低的字体列表。字体的选定*不是*在发现用户计算机上安装的列表中的第一个字体时停止。相反，对字体的选择是*逐字*进行的。也就是说即使某个字符周围都在某个字体中可以显示，但该字符在当前的字体文件中没有适合的图形，那么会继续尝试列表中靠后的字体。当一个字体只在某些特定的[样式](/zh-CN/docs/Web/CSS/font-style)、[变体](/zh-CN/docs/Web/CSS/font-variant)或[大小](/zh-CN/docs/Web/CSS/font-size) 下有效时，这些属性的值也可能对字体族的选择造成影响。
```